keyboards/splitkb/kyria/keymaps/default/keymap.c

```diff
@@ -15,7 +15,16 @@
  */
 #include QMK_KEYBOARD_H
 
-enum layers { _QWERTY = 0, _DVORAK, _COLEMAK_DH, _NAV, _SYM, _FUNCTION, _ADJUST };
+enum layers {
+    _QWERTY = 0,
+    _DVORAK,
+    _COLEMAK_DH,
+    _NAV,
+    _SYM,
+    _FUNCTION,
+    _ADJUST
```

Comment:
This should prevent it from reformatting the enum, as well.
```suggestion
    _ADJUST,
```

Don't ask me why.... clang-format is ... weird.